.github/workflows/reusable_dockerfile_pipeline.yml

```diff
@@ -294,4 +308,4 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           file: ${{ inputs.dockerfile }}
-          build-args: ${{ inputs.buildArgs }}
+          build-args: ${{ steps.convert_args.outputs.converted }}
```

Comment:
if `steps.convert_args.outputs.converted` is unset, it will resolve to an empty string which is valid default behaviour. So no additional checks are required